src/index.ts

```diff
@@ -30,6 +32,8 @@ export function createIPXHandler ({
   }
   const handler: Handler = async (event, _context) => {
     const host = event.headers.host
+    event.headers['x-forwarded-proto'] = X_FORWARDED_PROTO_VALUES.includes(event.headers['x-forwarded-proto'].toLowerCase()) ? event.headers['x-forwarded-proto'] : 'http'
+
     const protocol = event.headers['x-forwarded-proto'] || 'http'
```

Comment:
seems like a complex way of handling this, wdyt about this?
```suggestion
    const protocol = event.headers['x-forwarded-proto'] === "https" ? "https" : "http"
```

Reply:
Yeah, that makes sense. I think my brain was laser focused on sanitizing the header. Simplicity ftw.